script.js

```diff
@@ -136,12 +137,34 @@
         validateElement();
     };
 
+    var buildUrl = function (id) {
```

Comment:
See above. It hink it would be better if the server could handle this to avoid the extra header code.